src/install-edge-runtime.sh

```diff
@@ -49,43 +49,28 @@ function install_edge_runtime() {
 
     # create .toml from template
     log_info "Create instance configuration .toml from template."
-    cp /etc/aziot/config.toml.edge.template /etc/aziot/config.toml 2>>$STDERR_REDIRECT 1>>$STDOUT_REDIRECT
-    exit_code=$?
-    if [[ $exit_code != 0 ]];
-    then
-        log_info "'cp /etc/aziot/config.toml.edge.template /etc/aziot/config.toml' returned %d" $exit_code
-        exit ${EXIT_CODES[11]}
-    fi
 
     local SCOPE_ID=$1
     local REGISTRATION_ID=$2
     local SYMMETRIC_KEY=$3
 
     log_info "Set DPS provisioning parameters."
-    sed -i '/## DPS provisioning with symmetric key/,/## DPS provisioning with X.509 certificate/c\
-## DPS provisioning with symmetric key\
-[provisioning]\
-source = "dps"\
-global_endpoint = "https://global.azure-devices-provisioning.net"\
-id_scope = \"'$SCOPE_ID'\"\
-\
-[provisioning.attestation]\
-method = "symmetric_key"\
-registration_id = \"'$REGISTRATION_ID'\"\
-\
-symmetric_key = { value = \"'$SYMMETRIC_KEY'\" }                                                                         # inline key (base64), or...\
-# symmetric_key = { uri = "file:///var/secrets/device-id.key" }                                                          # file URI, or...\
-# symmetric_key = { uri = "pkcs11:slot-id=0;object=device%20id?pin-value=1234" }                                         # PKCS#11 URI\
-\
-## DPS provisioning with X.509 certificate\
-    '  /etc/aziot/config.toml 2>>$STDERR_REDIRECT 1>>$STDOUT_REDIRECT
-
-    exit_code=$?
-    if [[ $exit_code != 0 ]];
-    then
-        log_info "'sed ....'" $exit_code
-        exit ${EXIT_CODES[12]}
-    fi
+    local FILE_NAME="/etc/aziot/config.toml"
+    echo 'hostname = "'`hostname`'"' > $FILE_NAME
```

Comment:
>> $FILE_NAME [](http://example.com/codeflow?start=37&length=12)

if the file existed and was not empty, i think the content may potentially be corrupted. Do you think there is value in deletign the file if it already exists?